### ISSUE TITLE
Recompressing in .gz format after minor modification

### ISSUE THREAD

**Issue by blaisebarrette** [luc-github/ESP8266#477]
I use ESP3D with GRBL_ESP32 and I would like to add the following lines in the head section

```
<meta name="apple-mobile-web-app-capable" content="yes">
<meta name="apple-mobile-web-app-title" content="Cyclone">
<link rel="apple-touch-icon" href="/cyclone.png">
```

How do I go about recompressing it in .gz after my modification?

thanks!

**Comment by luc-github** [comment]
you means esp3d-webui ?
this time it is here : https://github.com/luc-github/ESP3D-WEBUI/wiki/Compilation---Development

ESP3D FW is a FW for ESP board as daughter boards
GRBL_ESP32 is GRBL FW with some code of ESP3D for the WIFI features
ESP3D-WEBUI is the html page used as frontend by ESP3D and GRBL_ESP32, and ESP3DLib used for Marlin on ESP32 Boards

I hope it clarify ^_^

**Comment by blaisebarrette** [comment]
It's a jungle out there! Hahaha!!! Ok then, yes I meant esp3D-webui.
The index.html.gz file found in the data folder that comes with the firmware GRBL_ESP32.

Can I uncompress the index.html file, add my few lines of code and simply reccmpress it back to it's .gz format? It's the recompressing bit I don't know how to do.

Thanks!

**Comment by luc-github** [comment]
yes you can just use gzip to recompress with default value , NB: gzip is not zip

**Comment by blaisebarrette** [comment]
OK! Thanks! I Kan manage that!
You've been very patient and helpful!
Many thanks!

- Blaise

**Comment by blaisebarrette** [comment]
It worked and I just wanted to say that it looks awesome as a webapp on an iPad!!!
If you haven't tried it, you should! It kick's ass!!!

**Comment by luc-github** [comment]
I do not have ipad, can you share the screenshot?
so adding the meta and the link have big impact on ipad?

**Comment by blaisebarrette** [comment]
Works with iPhone as well. Any iOS device. 
I made a short video so you ca see how it behaves.  

https://www.dropbox.com/sh/c6q9o0jbpqv208c/AAB6C015hJMZNf36A5QWFhV5a?dl=0

And here is a link where you will find all the info on what I did. It couldn't be simpler!

https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html

Tanks again!
